src/privacy.js

```diff
@@ -44,8 +44,7 @@ export async function shouldCollectTelemetry(method) {
  * @return {boolean}
  */
 export async function shouldUpdatePrices() {
-  // TODO (bdanforth): Add private browsing check per #177
-  return true;
+  return !isActiveWindowPrivate();
```

Comment:
`isActiveWindowPrivate` returns a promise, so we need to `await` it. This breaks the check currently and price updates never happen.

Reply:
If I add `await` here, I get an eslint error: `no-return-await`, Redundant user of `await` on a return value.

Reply:
That's for:

```js
return await !isActiveWindowPrivate();
```

Which isn't correct; `isActiveWindowPrivate()` returns a promise value:

```js
return await !(new Promise());
```

And promises are not falsy so:

```js
return await false;
```

And since `await` is actually valid with non-Promise values (it resolves to the value immediately) it's not a syntax error, and the linter just reminds you you don't need to await it.

The await we need is before the `!`:

```suggestion
  return !(await isActiveWindowPrivate());
```

Then we're actually running the `!` on the boolean value that's returned, rather than the Promise value.

Reply:
Hmm, something like this is correctly returning true:
```
/**
 * Determine if a background price update should start
 * @return {boolean}
 */
export async function shouldUpdatePrices() {
  return !(await isActiveWindowPrivate());
}
```

I will update `shouldExtract` similarly.